RFC_PROCESS.md

```diff
@@ -0,0 +1,39 @@
+# RFC Process
+
+## What's an RFC?
+
+The name itself is a reference to the IETF's Request For Comments process, and basically involves a document or series of documents which are drafted, reviewed, and eventually ratified (approved) by the Ayo.js community through discussion among those interested.
+
+An RFC can extend, modify, or alter any part of the Ayo.js project's code or documents, whether or not they've been previously documented. This includes proposing new features, proposing major breaking changes, changes to the Code of Conduct and other community policies, changes to the governance structure of the project, and anything else that affects a significant part of the community. It can also propose entirely new policies and community agreements.
+
+## How do I create an RFC?
+
+* Go to https://github.com/ayojs/ayo
+* Create a PR with a new RFC document:
+  * Must be in `rfcs`.
+  * Must be an `.md` file.
+  * The first line should be `# RFC - <title of RFC>`
+* Tag at least one member of any relevant working groups. If you're not sure who would be responsible for reviewing your RFC, let the triage process take care of it for you.
+* For small changes or moderate policy documentation changes, you may include the actual patch along with the main RFC. For example, if you're rewording something in the CoC, it might be appropriate to patch the CoC in the same PR as the RFC submission.
+
+## How does review work?
+
+All comments on a particular RFC should be done directly on the PR: An RFC will not be accepted until relevant reviewers are satisfied with changes. Members of the working groups the RFC falls under hold ultimate authority over whether or not to accept an RFC, if there's ever conflict about it, and may resolve that using their own governance and decision-making rules.
```

Comment:
s/working groups/teams

Reply:
this sentence is structured in a weird way that makes it hard to read, but i can't put my finger on it??